test/mri/ruby/test_pack.rb

```diff
@@ -857,4 +860,20 @@ def test_unpack1
     assert_equal "hogefuga", "aG9nZWZ1Z2E=".unpack1("m")
     assert_equal "01000001", "A".unpack1("B*")
   end
+
+  def test_pack_infection
```

Comment:
fix for pack/unpack tainting is already on master: https://github.com/jruby/jruby/commit/1d1bb5ddf65b320efc4161f20e1ba8338e4de986
... but you will need to tag this one since JRuby does not do much with 'P' (or 'p')

Reply:
I think @enebo was looking into this...maybe it doesn't need to be done.